apps/nextjs-app-router/src/app/[lang]/[[...path]]/page.tsx

```diff
@@ -4,6 +4,8 @@ import { getSiteVersion } from '@makeswift/runtime/next/server'
 import { notFound } from 'next/navigation'
 import { Page as MakeswiftPage } from '@makeswift/runtime/next'
 
+import '@/makeswift/components'
```

Comment:
This one shouldn't be needed, we need this import in only two places: root layout and Makeswift provider component file.